example/lib/main.dart

```diff
@@ -149,6 +169,57 @@ class _MyAppState extends State<MyApp> {
                   style: TextStyles.styleButton,
                 ),
               ),
+              TextButton(
+                onPressed: () {
+                  returnLastCoordinates();
+                },
+                child: Column(
+                  children: [
+                    Text(
+                      Strings.textReturnLastCoordinates,
+                      style: TextStyles.styleButton,
+                    ),
+                    StreamBuilder<dynamic>(
+                      stream: PlugInKotlin.locationEventStream,
+                      builder: (context, snapshot) {
+                        if (snapshot.data != null) {
+                          _coordinates = snapshot.data;
+                          if (!_coordinates.containsKey(Strings.status)) {
+                            _latitude = _coordinates[Strings.latitude]
+                                .toStringAsFixed(Numbers.numberCoordinates);
+                            _longitude = _coordinates[Strings.longitude]
+                                .toStringAsFixed(Numbers.numberCoordinates);
+                          } else {
+                            _latitude = Strings.noLatitude;
+                            _longitude = Strings.noLongitude;
+                          }
```

Comment:
You have a lot of logic here. An option is to create a class that hold latitude and longitude, and a method that set this two values as parameter of this method is snapshot.data

Reply:
snapshot.data != null can be replace with snapshot.hasData?

Reply:
done